utils/misc/misc_test.go

```diff
@@ -674,3 +674,38 @@ func TestMapLookup(t *testing.T) {
 	}
 	require.Nil(t, MapLookup(m, "foo"))
 }
+
+func TestGetDiskUsage(t *testing.T) {
+	initMisc()
+	// Create a temp file
+	tmpDirPath, err := CreateTMPDIR()
+	require.NoError(t, err)
```

Comment:
Please prefer using t.TempDir() which is different for every test execution and automatically cleaned up.
```suggestion
	tmpDirPath := t.TempDir()
```